.github/workflows/node.js.yml

```diff
@@ -3,7 +3,9 @@ name: CI
 on:
   pull_request:
   push:
-    branches-ignore: "dependabot/**"
+    branches-ignore:
+      - main
```

Comment:
Why ignore `main`? And what about `3.x-stable`?

Reply:
This was temporary. No longer ignoring main.